### ISSUE TITLE
Error with configuration

### ISSUE THREAD

**Issue by lucasbaylon** [ovflowd/chocolatey#388]
Hi

I configured the .env file and the file chocolatey.php and I have this error that appears, have you got an idea?

"Warning. Please read Chocolatey Installation Tutorial. You need configure your servers DocumentRoot to the public folder of this CMS. Thanks!"

**Comment by berkibap** [comment]
Have you ran composer install?

**Comment by lucasbaylon** [comment]
No, I do not know what it is, I've seen people talking about it, but I do not know what it's all about

**Comment by berkibap** [comment]
Read the newest wiki installation, it should give you instructions.

**Comment by lucasbaylon** [comment]
this one ? https://github.com/ovflowd/chocolatey/wiki/Installing-Chocolatey

**Comment by berkibap** [comment]
https://github.com/ovflowd/chocolatey/wiki/New-Tutorial

**Comment by lucasbaylon** [comment]
i'm not in localhost

**Comment by ovflowd** [comment]
Hey @ZaRToP the version of Chocolate you're using isn't supported. Just download this one: https://github.com/ovflowd/chocolatey/releases/tag/3.000N
